test/test-browser.js

```diff
@@ -18,17 +18,20 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 /* global window */
-if (window.browserTestDriver_finish) {
-  // Is automated test, add BrowserTestDriver hooks
-  const test = require('tape');
-  test.onFailure(window.browserTestDriver_fail);
-  test.onFinish(window.browserTestDriver_finish);
-}
+const probe = require('probe.gl'); // ensure probe gets access to pure console first
+const test = require('tape');
+
+let failed = false;
+test.onFailure(() => {
```

Comment:
Ah, this works around the false positives? Nice. 

Needs to be done in all frameworks though, maybe such logic could be inside the `browserTestDriver_fail();`? Or maybe that would be too clever?

Reply:
`browserTestDriver_fail` is actually executed on the node side. For the browser coloring we have to implement something here.